static/src/javascripts/projects/common/modules/identity/email-preferences.js

```diff
@@ -0,0 +1,141 @@
+// This is a workaround for the email preferences page https://profile.thegulocal.com/email-prefs
+// We want to submit subscribe/unsubscribe requests without a full page refresh
+// Hopefully this will be short-lived; if it is still alive in 2017, git blame and cry
+
+define([
+    'bean',
+    'qwery',
+    'reqwest',
+    'fastdom',
+    'common/utils/$'
+], function (
+    bean,
+    qwery,
+    reqwest,
+    fastdom,
+    $
+) {
+    function reqwestEmailSubscriptionUpdate(buttonEl) {
+        bean.on(buttonEl, 'click', function () {
+            buttonEl.disabled = true;
+            buttonEl.innerHTML = 'Loading...';
+            var formQueryString = generateFormQueryString(buttonEl);
+            reqwest({
```

Comment:
reqwest has contentType set in the defaultHeaders:
        `'contentType': 'application/x-www-form-urlencoded'`

So leaving it blank means the POST request contains:

<img width="422" alt="picture 157" src="https://cloud.githubusercontent.com/assets/8607683/17437322/033cd72c-5b15-11e6-8a83-226a45d87254.png">